common/src/main/java/dev/latvian/mods/kubejs/BuilderBase.java

```diff
@@ -45,7 +45,7 @@ public BuilderBase(ResourceLocation i) {
 	public abstract T createObject();
 
 	public final BuilderBase<T> type(String type) {
-		throw new RuntimeException("type(type: string) is no longer supported! Use event.create(id: string, type: string) now!");
+		throw new RuntimeException("type(type String) is no longer supported! Use event.create(id String, type String) now!");
```

Comment:
If we're gonna go for java-esque syntax, we should go for **java-esque** syntax (i.e. `String type`)

Reply:
Makes sense, fixed!